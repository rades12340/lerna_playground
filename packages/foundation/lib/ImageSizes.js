const imageSizes = {
    xs: 'xs',
    sm: 'sm',
    md: 'md',
    lg: 'lg',
    xl: 'xl',
    xxl: 'xxl',
};
export default Object.freeze(imageSizes);
