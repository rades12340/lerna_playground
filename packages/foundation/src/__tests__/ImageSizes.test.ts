import ImageSizes from "../ImageSizes"

test('snapshot of image sizes', () => {
    expect(ImageSizes).toMatchSnapshot()
})