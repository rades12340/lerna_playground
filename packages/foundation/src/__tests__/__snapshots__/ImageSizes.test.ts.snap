// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of image sizes 1`] = `
Object {
  "lg": "lg",
  "md": "md",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
}
`;
